package Auto_Secure.Client.Interface.Exceptions;

public class RecursoNaoEncontradoException extends RuntimeException {
    public RecursoNaoEncontradoException(String s) {
    }
}
